server/shared/mail/templates/comment.mjml

```diff
@@ -3,33 +3,52 @@
   <mj-body padding="10px">
     <mj-include path="./components/header.mjml"/>
     <mj-section
-      padding="0 10px"
+      padding="0 10px 20px"
       text-align="left">
-      <mj-column border-bottom="1px solid #ccc">
+      <mj-column>
         <mj-text
           mj-class="content"
           padding-left="0"
           color="#444"
           font-size="16px">
-          <b>{{author.label}}</b> left a comment on {{repositoryName}}, {{topic}}.
-        </mj-text>
-      </mj-column>
-    </mj-section>
-    <mj-section padding="20px 10px">
-      <mj-column
-        background-color="#f1f1f1"
-        border-radius="8px">
-        <mj-text
-          mj-class="content"
-          padding="20px"
-          color="#444"
-          font-size="16px">
-          {{content}}
+          <b>{{author.label}}</b> left a comment on <b>{{repositoryName}}</b>, <b>{{topic}}</b>.
         </mj-text>
       </mj-column>
     </mj-section>
     <mj-section padding="0 10px">
-      <mj-column border-top="1px solid #ccc">
+			<mj-column border-bottom="2px dashed #d2d2d2">
+        <mj-table padding="10px">
```

Comment:
See if there is a better way of implementing this layout

Reply:
Initially I have tried using the common mjml tags (mj-section, mj-column) but afaik it is not possible to achieve this exact layout.